lib/spreadsheet_architect/utils.rb

```diff
@@ -150,6 +150,10 @@ def self.get_options(options, klass)
         end
       end
 
+      unless options[:escape_formulas].class.in?([TrueClass, FalseClass, Array])
```

Comment:
This should simply be `if options[:escape_formulas].nil?`

Reply:
escape_formulas, needs to be added to `ALLOWED_OPTIONS` too. It should also be included in the project-wide/class-wide options too.

Reply:
Also I dont know if that we should default to true for escape formulas, this would likely be a breaking change for some users. I think for legacy reasons maybe we should default to false.